src/core/agent/services/identifierService.ts

```diff
@@ -376,14 +377,24 @@ class IdentifierService extends AgentService {
       );
       delegateAid = { state: delegator.response } as Aid;
     }
+    if (!threshold) {
+      threshold = otherAids.length + 1;
+    }
+
     const signifyName = utils.uuid();
     const result = await this.agent.modules.signify.createMultisig(
       ourAid,
       otherAids,
       signifyName,
-      delegateAid
+      delegateAid,
+      threshold
     );
     const multisigId = result.op.name.split(".")[1];
+    //this will be updated once the operation is done
+    let isPending = true;
+    if (result.op.done || threshold === 1) {
+      isPending = false;
+    }
```

Comment:
Tests should check that `isPending = true` but `isPending = false` if threshold was 1, or if the result.op was complete already and threshold didn't equal 1.